metadata-ingestion/src/datahub/configuration/common.py

```diff
@@ -158,6 +162,26 @@ class IgnorableError(MetaError):
     """An error that can be ignored."""
 
 
+@runtime_checkable
+class ExceptionWithProps(Protocol):
+    def get_telemetry_props(self) -> Dict[str, Any]:
+        ...
```

Comment:
Does this mean... pass?

Reply:
yes it's a python thing

`...` is convention for typing-related stuff